crm_request/models/request.py

```diff
@@ -101,3 +101,15 @@ def message_post(self, **kwargs):
                     'crm_request', 'stage_wait_customer')[1]
 
         return result
+
+    @api.multi
+    def write(self, values):
+        # Get all followers before change
```

Comment:
You should first check if it is in the "assign to" action.

for record in self:
    if values.get('user_id')  and record.user_id is None:
        ...